src/test/java/de/orchound/gameoflife/game/BoardTest.java

```diff
@@ -39,4 +39,22 @@ void testUpdate() {
 		assertFalse(board.target[1][2]);
 		assertFalse(board.target[3][2]);
 	}
+
+	@Test
+	void testUpdateWrapBoard() {
```

Comment:
```suggestion
	void testHorizontalWrapping() {
```

Also write another test `testVerticalWrapping()` and use different widths and heights for the test board (`Board board = new Board(20, 10);`).
That way the error in `Board.java` would've been detected.